griptape/tools/google_sheets/tool.py

```diff
@@ -0,0 +1,436 @@
+                )
+    
+                for file in response.get("files", []):
+                    spreadsheets.append(InfoArtifact(f"Spreadsheet ID: {file['id']}, Name: {file['name']}"))
+    
+                page_token = response.get("nextPageToken")
+            return ListArtifact(spreadsheets)
+    
+        except HttpError as e:
+            return ErrorArtifact(f"error listing spreadsheet due to http error: {e}")
+        except MalformedError as e:
+            return ErrorArtifact(f"error listing spreadsheet due to malformed credentials: {e}")
+        except Exception as e:
+            return ErrorArtifact(f"error listing spreadsheets: {e}")
+
+    @activity(
+        config={
+            "description": "Can be used to creates a new Google Sheet with the specified title.",
+            "schema": Schema(
+                {
+                    Literal(
+                        "title", description="The title of the new spreadsheet"
+                    ): str
+                }
+            ),
+        }
+    )
+    def create_spreadsheet(self, params: dict) -> InfoArtifact | ErrorArtifact:
+        from googleapiclient.errors import HttpError
+        from google.auth.exceptions import MalformedError
+
+        values = params["values"]
+        title = values.get("title")
+
+        try:
+            service = self._build_client(
+                scopes=self.SHEETS_SCOPES,
+                service_name="sheets",
+                version="v4",
+                owner_email=self.owner_email,
+            )
+
+            spreadsheet_body = {"properties": {"title": title}}
+
+            spreadsheet = (
+                service.spreadsheets().create(body=spreadsheet_body).execute()
+            )
+            return InfoArtifact(
+                f"Spreadsheet created with ID: {spreadsheet['spreadsheetId']}"
+            )
+
+        except HttpError as e:
+            return ErrorArtifact(
+                f"error creating spreadsheet due to http error: {e}"
+            )
+        except MalformedError as e:
+            return ErrorArtifact(
+                f"error creating spreadsheet due to malformed credentials: {e}"
+            )
+        except Exception as e:
+            return ErrorArtifact(f"error creating spreadsheets: {e}")
+
+    @activity(
+        config={
+            "description": "Can be used to downloads multiple spreadsheets based on provided file paths",
+            "schema": Schema(
+                {
+                    Literal(
+                        "file_paths",
+                        description="List of file paths to the spreadsheets.",
+                    ): list[str],
+                    Literal(
+                        "mime_type",
+                        description="The MIME type for the file format to export to.",
+                    ): str,
+                }
+            ),
+        }
+    )
+    def download_spreadsheets(self, params: dict) -> ListArtifact | ErrorArtifact:
```

Comment:
I don't think we need this activity since we already have `download_files` in the drive tool.

Reply:
Yup, removed it